packages/website/src/components/lib/jsonSchema.ts

```diff
@@ -18,30 +23,18 @@ const defaultRuleSchema: JSONSchema4 = {
  */
 export function getRuleJsonSchemaWithErrorLevel(
   name: string,
-  ruleSchema: JSONSchema4 | JSONSchema4[],
+  ruleSchema: JSONSchema4 | readonly JSONSchema4[],
 ): JSONSchema4 {
-  if (Array.isArray(ruleSchema)) {
+  if (isArray(ruleSchema)) {
     return {
       type: 'array',
       items: [defaultRuleSchema, ...ruleSchema],
       minItems: 1,
       additionalItems: false,
     };
   }
-  // TODO: delete this once we update schemas
-  // example: ban-ts-comment
-  if (Array.isArray(ruleSchema.prefixItems)) {
```

Comment:
after infinite loop is fixed, we should verify if eslint is not using it:

```
monaco.languages.json.jsonDefaults.diagnosticsOptions.schemas.filter(item => item.schema.prefixItems)
```

Reply:
We are not using it, nor is eslint.
Its not a valid Json Schema v4 property.

It wasn't caught before because the JSONSchema type had a `[k: string]: any` signature that allowed additional, incorrect properties.
I patched the package to remove that signature (hence I found this error too).

Using `prefixItems` was what caused a few of our rules to not validate their options (because v6 of avj only supports v4 of json schema)

Reply:
yes i just validated it and its no longer used, great job 👍

Reply:
> Using prefixItems was what caused a few of our rules to not validate their options (because v6 of avj only supports v4 of json schema)

ajv supports other versions of schema but eslint is forcing it to use v4, 
https://github.com/eslint/eslint/blob/1fea2797801a82a2718814c83dad641dab092bcc/lib/shared/ajv.js#L11-L12